ko-archetype/src/main/resources/archetype-resources/js/pom.xml

```diff
@@ -100,9 +102,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.openjdk.nashorn</groupId>
-            <artifactId>nashorn-core</artifactId>
-            <version>\${nashorn.version}</version>
+            <groupId>org.graalvm.js</groupId>
+            <artifactId>js-scriptengine</artifactId>
```

Comment:
- nashorn no longer works with JDK25!?
- that's surprising
- what's the failure?